### DISCUSSION TITLE
How do you feel about EUPL 1.2?

### DISCUSSION THREAD

**Discussion by TheGuyWhoo** [DISTRHO/Cardinal#239] [General]
I'd really like these to be added:

https://github.com/mgunyho/PdArray

but unsure if the license is a blocker.

**Comment by falkTX** [comment]
from what I can tell, EUPL is fine and 100% compatible with GPL.
It is basically GPL adapted to the european union, with the nice bonus of having a legally-binding official translation to quite a few languages.

I quite like the license, and might apply it to future projects.
